native/common/include/jpype.h

```diff
@@ -12,16 +12,15 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
-   
+
  *****************************************************************************/
 #ifndef _JPYPE_H_
-#define _JPYPE_H_ 
-
-// Define this to generate the trace calls
+#define _JPYPE_H_
 
-// Define this to make the trace calls do their output. 
-//    use "setup.py --enable-tracing build" to enable 
-//#define JP_TRACING_ENABLE
+#ifdef __GNUC__ // Then this must mean a variant of GCC on win32 ...
```

Comment:
This comment seems wrong here, since it is outside of the WIN32 definition check.

Reply:
I will look this over.  I was mainly deleting old stuff and after the deletion the remaining was left like this.  Perhaps the editor ate something.

Reply:
Actually it appears that it was just me cutting and pasting to try to disable a warning.  I will clean this section up.  Near as I can tell most of it is cruft.